admin_manual/release_notes.rst

```diff
@@ -18,6 +19,111 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.10_release_notes_label:
+
+Changes in 10.0.10
+--------------------------
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.0.10 that need your attention. You can also read `the full ownCloud Server changelog`_ for further details on what has changed.
+
+Official PHP 7.2 support
+~~~~~~~~~~~~~~~~~~~~~~~~
+After announcing the future deprecation of PHP 5.6 and 7.0 with the `10.0.8 release <https://doc.owncloud.com/server/10.0/admin_manual/release_notes.html#php-5-6-deprecation>`_, ownCloud Server now follows up with official PHP 7.2 support. The Server Core and all apps maintained by ownCloud have received a full QA cycle and are proven to work reliably. With this step ownCloud Server is also being prepared for PHP 7.3 which is `scheduled to become available by the end of 2018 <https://wiki.php.net/todo/php73>`_. When still operating on 5.6 or 7.0, please plan an upgrade to 7.2 soon. See the `system requirements in the ownCloud Documentation <https://doc.owncloud.com/server/latest/admin_manual/installation/system_requirements.html#officially-recommended-supported-options>`_.
+
+Note: With PHP 7.2 some extensions have changed. When it's not installed yet, you need to additionally install ``php-openssl``. See `#30337 <https://github.com/owncloud/core/issues/30337>`_ for more information.
+
+New local user creation flow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In previous versions administrators created local users by entering a *Username* and a *Password*. In many cases this is undesirable as administrators set the password for new users and need to provide it on another communication channel. For this reason the local user creation flow has been changed to expect a *Username* and an *E-Mail address* which will be used to send an activation link to new users. This way user creation is easier and more secure as new users are informed automatically and can choose a password in self-service. For cases where administrators want to set the iniial password it's possible to deviate from the default by setting the option "Set a password for new users" on the bottom left settings cog. The former option "Send email to new users" has been removed as this change made it obsolete.
+
+HTTP API for Search
+~~~~~~~~~~~~~~~~~~~
+ownCloud Server 10.0.10 introduces a HTTP API for the search functionality. It enables querying for search terms on the server and delivering results via HTTP (WebDAV). With upcoming releases the ownCloud Clients will make use of it to be able to search contents on the server without the need of having them available locally. 
```

Comment:
- s/for the search functionality/for search functionality/g
- Should "It enables querying for search terms on the server and delivering results via HTTP (WebDAV)" be written as "_It enables the use of search terms to query the server and the delivery of search results via HTTP (WebDAV)_"?
- I'd suggest rewriting the final sentence as follows: "_In upcoming releases, ownCloud Clients will make use of it to search contents on the server, without the need to have them available locally._"